packages/create-next-app/index.ts

```diff
@@ -99,70 +98,11 @@ async function run(): Promise<void> {
     process.exit(1)
   }
 
-  if (!program.example) {
-    const template = await prompts({
-      type: 'select',
-      name: 'value',
-      message: 'Pick a template',
-      choices: [
-        { title: 'Default starter app', value: 'default' },
-        { title: 'Example from the Next.js repo', value: 'example' },
-      ],
-    })
-
-    if (!template.value) {
-      console.log()
-      console.log('Please specify the template')
-      process.exit(1)
-    }
-
-    if (template.value === 'example') {
-      let examplesJSON: any
-
-      try {
-        examplesJSON = await listExamples()
-      } catch (error) {
-        console.log()
-        console.log(
-          'Failed to fetch the list of examples with the following error:'
-        )
-        console.error(error)
-        console.log()
-        console.log('Switching to the default starter app')
-        console.log()
-      }
-
-      if (examplesJSON) {
-        const allChoices = examplesJSON.map((example: any) => ({
-          title: example.name,
-          value: example.name,
-        }))
-        // The search function built into `prompts` isn’t very helpful:
-        // someone searching for `styled-components` would get no results since
-        // the example is called `with-styled-components`, and `prompts` searches
-        // the beginnings of titles.
-        const nameRes = await prompts({
-          type: 'autocomplete',
-          name: 'exampleName',
-          message: 'Pick an example',
-          choices: allChoices,
-          suggest: (input: any, choices: any) => {
-            const regex = new RegExp(input, 'i')
-            return choices.filter((choice: any) => regex.test(choice.title))
-          },
-        })
-
-        if (!nameRes.exampleName) {
-          console.log()
-          console.log(
-            'Please specify an example or use the default starter app.'
-          )
-          process.exit(1)
-        }
-
-        program.example = nameRes.exampleName
-      }
-    }
+  if (program.example === true) {
+    console.error(
+      'Please provide an example name or url, otherwise remove the example option.'
+    )
+    process.exit(1)
```

Comment:
Out of paranoia can we return after this?
```suggestion
    process.exit(1)
    return
```